packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx

```diff
@@ -22,13 +37,40 @@ export const DashboardExtension = ({
   const location = useLocation();
   const navLink = useMemo(() => decodeURIComponent(last(location.pathname.split('/'))), [location.pathname]);
 
+  type MenuTitle = keyof typeof MenuIcons;
+
+  const MenuIcons = {
```

Comment:
Thanks @Munyua123 for this, what will be the approach for adding icons for packages that do not live in the patient chart ?

Reply:
@CynthiaKamau The approach may be adding the icons in their respective packages.